librz/core/core.c

```diff
@@ -2446,3 +2446,36 @@ RZ_API RzCmdStatus rz_core_core_plugins_print(RzCore *core, RzCmdStateOutput *st
 	rz_cmd_state_output_array_end(state);
 	return RZ_CMD_STATUS_OK;
 }
+
+/**
+ * \brief Filters the given output string based on the provided filter.
+ *
+ * \param output The output string to be filtered.
+ * \param filter The filter string to be used for filtering the output.
+ * \return char* The filtered output string. The caller is responsible for freeing this string.
+ */
+RZ_API char *rz_core_filter_string_output(RZ_NONNULL const char *output, const char *filter) {
+	rz_return_val_if_fail(output, NULL);
+	char *filtered_output = NULL;
+	char *output_copy = rz_str_dup(output);
+	RzList *lines = rz_str_split_list(output_copy, "\n", 0);
+	RzListIter *iter;
+	char *line;
+	rz_list_foreach (lines, iter, line) {
+		if (strstr(line, filter)) {
+			if (filtered_output) {
+				filtered_output = rz_str_append(filtered_output, "\n");
+			} else {
+				filtered_output = rz_str_dup("");
+			}
+			filtered_output = rz_str_append(filtered_output, line);
+		}
+	}
+	RZ_FREE(output_copy);
+	return filtered_output;
+}
+
+RZ_API char *rz_core_get_afb_output(RZ_NONNULL RzCore *core) {
```

Comment:
Remove this, and use the function instead of calling the command, it also should happen in the `visual.c` instead.